graphql-cli/src/main/java/io/ballerina/graphql/exception/ConfigTypesGernerationException.java

```diff
@@ -0,0 +1,49 @@
+/*
+ *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 *  Copyright (c) 2022, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Reply:
Updated in https://github.com/ballerina-platform/graphql-tools/commit/4c09a3ac3dfec85f885db25f7872508822d2e17d